compiler/parser/parser.cpp

```diff
@@ -891,7 +895,7 @@ int dynoRealizeErrors(void) {
     }
     gDynoErrorHandler->clear();
   }
-  return ret;
+  return numErrors;
```

Comment:
Now it seems weird that this function returns an error count because there are 2 counts -- unique errors and total errors. Can we make it return a `bool` instead for "an error was reported" ?